dotCMS/src/main/java/com/dotcms/rendering/velocity/services/PageRenderUtil.java

```diff
@@ -387,20 +387,40 @@ private boolean needParseContainerPrefix(final Container container, final String
      * @throws DotSecurityException
      * @throws DotDataException
      */
-    private String getRelativePathFromSite(final FileAssetContainer container)  {
+    private String getRelativePathFromSite(final FileAssetContainer container) {
         return this.site.getIdentifier().equals(container.getHost().getIdentifier()) ?
                 container.getPath() :
                 FileAssetContainerUtil.getInstance().getFullPath(container);
     }
 
-    @Nullable
     private Contentlet getContentlet(final PersonalizedContentlet personalizedContentlet) {
+
+        return (Config.getBooleanProperty("DEFAULT_CONTENT_TO_DEFAULT_LANGUAGE", false)) ?
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Useless parentheses.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5316867)

Reply:
I am not sure if this change is needed, I think the 'DEFAULT_CONTENT_TO_DEFAULT_LANGUAGE' property is working, here we have a test that check it

https://github.com/dotCMS/core/blob/master/dotCMS/src/integration-test/java/com/dotcms/rendering/velocity/services/HTMLPageAssetRenderedTest.java#L659